lib/redux/actions/sync.js

```diff
@@ -89,11 +132,15 @@ export function setForgotPasswordUrl(url) {
   };
 }
 
-export function setPage(page, actionSource = actionSources[actionTypes.SET_PAGE]) {
+export function setPage(page, actionSource = actionSources[actionTypes.SET_PAGE], metric) {
+  let meta = {source: actionSource};
```

Comment:
I think you could actually use `const` here (and in the AirBnb ESLint config world it would be preferable) - because the `_.assign` doesn't reassign anything _to_ metric, just assigns new properties within it, which is allowed by `const`.